File: python/dictionaries.py
import time, hashlib

start = time.time()

prev = hashlib.md5(str(0).encode('utf-8')).hexdigest()
for i in range(1, 1000):
    dict = {}
    for j in range(1, 1000):
        dict[hashlib.md5(str(j).encode('utf-8')).hexdigest()] = j
    dict = sorted(dict)

end = time.time()
print("{0} sec.".format(end - start))

File: php/dictionaries.php
<?php

$start = microtime(true);

for ($i = 1; $i <= 1000; $i++) {
    $dict = [];
    for ($j = 1; $j <= 1000; $j++) {
        $dict[md5($j)] = $j;
    }

    ksort($dict);
}

$end = microtime(true);
printf('%f sec.', $end - $start);